src/styles/pages/blogPost.scss

```diff
@@ -31,7 +31,7 @@
 			}
 		}
 
-		@media only screen and (max-width: 1024px) {
+		@media screen and (width < 1024px) {
```

Comment:
Please don't use media query level 4 since they are only available on firefox right now.

Reply:
Yes, but I added a PostCSS plugin which converts them to regular media queries when building.